app/templates/_has_wants_selector.html

```diff
@@ -0,0 +1,182 @@
+{% macro ppe_row(ppe) %}
+<td><img class="itmimg" src="{{ ppe.img.decode() }}" alt="{{ ppe.desc }} mask image" width="150" /></td>
+<td>{{ ppe.sku }}</td>
+<td>{{ ppe.desc }}</td>
+<td>
+    {{ caller() }}
+</td>
+{% endmacro %}
+
+{% macro active_ppe_row(ppe) %}
+<!-- Since there can only be one answer, we guess what type of row this is.-->
+{% if ppe.wants|length > 0 %}
+    {% set count = ppe.wants[0].count %}
+{% elif ppe.has|length > 0 %}
+    {% set count = ppe.has[0].count %}
+{% else %}
+    {% set count = 0 %}
+{% endif %}
+{% call ppe_row(ppe) %}
+<span class="no-edit">{{ count }}</span>
+<input class="form-control" id="ppe-quantity-{{ppe.id}}" name="ppe-{{ppe.id}}" type="number" min="0" max="999999999" step="1" value="{{count}}" />
+<button class="btn btn-danger only-edit" onclick="document.getElementById('ppe-quantity-{{ppe.id}}').value = '0'; $(this).parent().parent().hide(); return false">
```

Comment:
On delete, we set the quantity to zero and hide the row so it looks like it went away.